KeyAuth.md

```diff
@@ -0,0 +1,15 @@
+# HTTPS with client certificate authentication
+
+To protect your openHAB instance in the public Internet network we can use client authenticate with SSL certificate feature.
+
+In order to prepare environment to work with this feature we must to do some below steps:
+
+## Own Certificate Authority
+TODO
```

Comment:
That would be great, I will go ahead and merge this and look forward to another PR for the readme!